transcendence-app/src/shared/filters/bad-request-transformation.filter.ts

```diff
@@ -0,0 +1,10 @@
+import { ArgumentsHost, BadRequestException, Catch } from '@nestjs/common';
+import { BaseWsExceptionFilter, WsException } from '@nestjs/websockets';
+
+@Catch(BadRequestException)
+export class BadRequestTransformationFilter extends BaseWsExceptionFilter {
```

Comment:
We use this to handle exceptions thrown by the validation pipes in the WebSocket context https://stackoverflow.com/a/63396944